ethereum/block/wait_for_receipt.go

```diff
@@ -16,8 +16,9 @@ import (
 
 const (
 	// in total, we will wait up to 1 hour for a transaction to be included
-	maxRetries           = 360
-	receiptSleepDuration = time.Second * 10
+	maxRetries = 360
+	// TODO: this should vary based on environment, can increase to 12s for mainnet.
+	receiptSleepDuration = time.Second * 2
```

Comment:
You could probably just leave it, unless the concern is that we'll generate too many queries to the ethereum endpoint. Polling every 2 seconds won't generate any kind of load on the swapd host. If you wanted to do it *without* the environment value, you could take the delta between the last two blocks. The code that is using it doesn't know how many seconds have passed since the last block was generated before calling sleep, so we probably don't want to wait the full inter-block delta anyways.

Reply:
I changed it because it was causing some unit tests to fail, since I didn't want a unit test waiting 10+ seconds for a receipt when it could have gotten it much faster

Reply:
I understand.  When I wrote "leave it" above, I was referring to the TODO to change the code yet again in the future. The code change you made is a good one.

Reply:
oh woops, sorry I misunderstood lol. cool, can remove the TODO then